satpy/tests/reader_tests/test_seviri_l1b_native.py

```diff
@@ -1206,6 +1230,7 @@ def test_get_dataset(self, file_handler):
                                       np.datetime64('1958-01-02 00:00:04')])
         xr.testing.assert_equal(dataset, expected)
         assert 'raw_metadata' not in dataset.attrs
+        assert file_handler.end_time == datetime(2006, 1, 1, 12, 27, 9, 304888)
```

Comment:
Can you please add a test for the `start_time` attribute as well?

Reply:
I think this is tested elsewhere. At least, the `start_time` property is called from the tests whereas (until now) `end_time` was not.

Reply:
I have now added an explicit test for this, better to test this directly rather than indirectly through some other test.